docs/api/05_subscriptions/create-subscription.mdx

```diff
@@ -131,23 +131,21 @@ import TabItem from '@theme/TabItem';
 ```json
 {
   "subscription": {
-    "customer_id": "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
+    "external_customer_id": "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
     "plan_code": "new_code",
-    "unique_id": "id",
     "name": "name",
-    "subscription_id": "sub id",
+    "external_id": "sub id",
     "billing_time": "anniversary"
   }
 }
 ```
 
 | Attributes | Type | Description |
 | -----------| -----| ----------- |
-| customer_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Customer unique identifier in your application |
+| external_customer_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Customer unique identifier in your application |
 | plan_code | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Code identifying the plan.<br></br>It must match the `code` property of one of the plans. |
-| unique_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique external identifier of the subscription. Used as an idempotency key. Can also be used for upgrades/downgrades. Default value if not provided: `customer_id`. |
+| external_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique external identifier of the subscription. Used as an idempotency key. Can also be used for upgrades/downgrades. Default value if not provided: `external_customer_id`. |
```

Comment:
I believe `external_id` is now required and the fallback has been removed (i.e. does not default to `external_customer_id` anymore).

Reply:
Thanks Mathieu - Updating the base branch about external id 👍